packages/main/src/plugin/tasks/task-manager.ts

```diff
@@ -27,10 +27,8 @@ import { IConfigurationRegistry } from '/@api/configuration/models.js';
 import type { NotificationTaskInfo, TaskInfo } from '/@api/taskInfo.js';
 import { ExperimentalTasksSettings } from '/@api/tasks-preferences.js';
 
-import taskManagerImage from '../../assets/tasks.Manager.webp';
```

Comment:
issue: you also need to remove tasks.Manager.webp file

Reply:
Good catch !